cfgov/cfgov/settings/base.py

```diff
@@ -60,12 +60,12 @@
     'legacy',
     'django_extensions',
     'reversion',
-    'tinymce'
```

Comment:
Jobmanager uses it as well for the legacy Django models, to edit the job HTML in the Django admin.